server/controllers/methods.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../dbconnection/schema.js'); //accese to your scheema;
+
+exports.addData = (req,res) => {
```

Comment:
my understanding is that req and res are used in server side calls, while db queries dont require them, but passes in a callback